egs2/aphasiabank/asr1/run.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set="train"
+valid_set="val"
+test_sets="test"
+include_control=false
+include_aphasia_type=true
+
+asr_tag=""
+if "${include_aphasia_type}"; then
+  asr_tag="train_asr_conformer_with_aphasia_type"
+fi
+
+# asr_config=conf/tuning/train_asr_transformer.yaml
+asr_config=conf/tuning/train_asr_conformer.yaml
+# asr_config=conf/tuning/train_asr_conformer_hubert_ll60k_large.yaml
+inference_config=conf/decode.yaml
+
+lm_config=conf/train_lm_transformer.yaml
+use_lm=false
+
+feats_normalize=global_mvn
+# feats_normalize=utt_mvn # if using HuBERT, https://github.com/espnet/espnet/issues/4006#issuecomment-1047898558
+
+# speed perturbation related
+# (train_set will be "${train_set}_sp" if speed_perturb_factors is specified)
+speed_perturb_factors="0.9 1.0 1.1"
+
+./asr.sh                                                                                                      \
+    --lang en                                                                                                 \
```

Comment:
The position of "\" is a bit weird from this view, as it is too far from the end of the line. Please double-check if the space symbol is correctly used.